pkg/apis/garden/v1beta1/conversion_test.go

```diff
@@ -0,0 +1,174 @@
+package v1beta1
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"time"
+
+	"github.com/gardener/gardener/pkg/apis/garden"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var _ = Describe("Conversion", func() {
+	var (
+		expirationDate = &metav1.Time{Time: time.Now().Add(time.Second * 20)}
+	)
+	Describe("#V1Beta1MachineImageToGardenMachineImage", func() {
+		It("external machine image should be properly converted to internal machine image", func() {
+			v1betaMachineImage := &MachineImage{
+				Name:    "coreos",
+				Version: "1.0.0",
```

Comment:
As you test this later you could store name and version in a variable (same for other tests, please check).